src/components/appheader.main.jsx

```diff
@@ -92,7 +92,9 @@ class AppHeaderMain extends React.Component {
           </div>
 
           <div className="cart-link-container">
-            <Link className="cart-link" to="/mybag" />
+            <Link className="cart-link" to="/mybag">
```

Comment:
I threw this in since the new version of Chrome we're using wasn't displaying our CSS when the UT was run. If that's still the case, this can stay as it's a good fallback to have text in place.